tests/hostlist.py

```diff
@@ -72,7 +72,7 @@ def test_simple_host(self):
 
     def test_aliases(self):
```

Comment:
Consider adding a test to cover the case we just fixed? aka Where one of the alias globs doesn't match any hosts, but its not an alias we care about